pkg/piecestore/rpc/server/server_test.go

```diff
@@ -25,18 +25,18 @@ import (
 	pb "storj.io/storj/protos/piecestore"
 )
 
-var tempDir string = path.Join(os.TempDir(), "test-data", "3000")
-var tempDBPath string = path.Join(os.TempDir(), "test.db")
+var tempDir = path.Join(os.TempDir(), "test-data", "3000")
+var tempDBPath = path.Join(os.TempDir(), "test.db")
 var db *sql.DB
 var s Server
 var c pb.PieceStoreRoutesClient
-var testId string = "11111111111111111111"
+var testID = "11111111111111111111"
```

Comment:
i would like to reiterate that globals make for brittle tests. these should be variables in a per-test struct somewhere